respect-app-compose/src/commonMain/kotlin/world/respect/app/view/clazz/detail/ClazzDetailScreen.kt

```diff
@@ -0,0 +1,265 @@
+                ?.lowercase()
+                ?.replaceFirstChar { it.uppercase() } ?: "Unknown"
+
+            ListItem(
+                modifier = Modifier
+                    .fillMaxWidth(),
+                leadingContent = {
+                    RespectPersonAvatar(
+                        name = pendingUser.givenName,
+                        modifier = Modifier.size(40.dp),
+                        fontScale = 1.0f
+                    )
+                },
+                headlineContent = {
+                    Text(text = pendingUser.givenName + "(" + roleName + ")")
+                },
+                trailingContent = {
+                    Row {
+                        Icon(
+                            imageVector = Icons.Outlined.CheckCircle,
+                            contentDescription = null,
+                            modifier = Modifier.size(24.dp),
+                        )
+                        Icon(
+                            imageVector = Icons.Outlined.Cancel,
+                            contentDescription = null,
+                            modifier = Modifier.size(24.dp),
+                        )
+                    }
+                }
+            )
+        }
+
+
+        item {
+            ListItem(
+                headlineContent = {
+                    Text(
+                        modifier = Modifier.padding(top = 24.dp),
+                        text = stringResource(
+                            resource = Res.string.teachers
+                        )
+                    )
+                },
+                trailingContent = {
+                    Icon(
+                        imageVector = Icons.Outlined.KeyboardArrowDown,
+                        contentDescription = null,
+                        modifier = Modifier.size(24.dp),
+                    )
+                }
+            )
+        }
+
+        item {
+            ListItem(
+                modifier = Modifier.clickable {
+                    onClickAddPersonToClazz()
+                },
+                leadingContent = {
+                    Icon(
+                        imageVector = Icons.Filled.Add,
+                        contentDescription = null
+                    )
+                },
+                headlineContent = {
+                    Text(
+                        text =
+                            stringResource(resource = Res.string.add_teacher)
+                    )
+                }
+            )
+        }
+
+
+        itemsIndexed(
+            uiState.listOfTeachers,
+            key = { index, teacher ->
+                teacher.sourcedId
+            }) { index, teacher ->
+            ListItem(
+                modifier = Modifier
+                    .fillMaxWidth(),
+                leadingContent = {
+                    RespectPersonAvatar(
+                        name = teacher.givenName,
+                        modifier = Modifier.size(40.dp),
```

Comment:
Let the defaults do their job - you are specifying the default size and fontscale.